### ISSUE TITLE
Predict model for evalute is Zero ?

### ISSUE THREAD

**Issue by NguyenThaiHoc1** [leeyeehoo/CSRNet-pytorch#79]
I have a problems, After I training but model predict=0.0, I don't know why,It's really tough to me.
I still try to change learning rate. But the problem don't handle 
Please help me
